packages/core/src/Cardano/types/Certificate.ts

```diff
@@ -80,20 +84,22 @@ export interface PoolRetirementCertificate {
 
 export interface StakeDelegationCertificate {
   __typename: CertificateType.StakeDelegation;
-  address: Address;
+  rewardAccount: RewardAccount;
   poolId: PoolId;
   epoch: Epoch;
 }
 
 export interface MirCertificate {
   __typename: CertificateType.MIR;
+  // Review: need to learn what this cert is and figure out if 'address' is actually an Address or a RewardAccount
```

Comment:
MIR = Move instantaneous rewards
This is a special mechanism that enables funds to be moved from reserves or the treasury into a _reward_ account. It was required as a way to enable rewards generated on side-chains, such as the Incentivized Testnet (ITN), to be applied to participants on the main chain.